plugins/core/jetbrains-community/resources/telemetryOverride.json

```diff
@@ -653,6 +653,10 @@
                 {
                     "type": "cwsprChatMessageId"
                 },
+                {
+                    "type": "cwsprChatUserIntent",
+                    "required": false
+                },
```

Comment:
should be pushing this into the common definitions

Reply:
yep, this was mentioned in my VSCode plugin change as well. Going to get out that change